modules/widgets/src/index.ts

```diff
@@ -2,19 +2,22 @@
 // SPDX-License-Identifier: MIT
 // Copyright (c) vis.gl contributors
 
-export {FullscreenWidget} from './fullscreen-widget';
-export {CompassWidget} from './compass-widget';
-export {ZoomWidget} from './zoom-widget';
-export {ScreenshotWidget} from './screenshot-widget';
+export {FullscreenWidget} from './widgets/fullscreen-widget';
+export {CompassWidget} from './widgets/compass-widget';
+export {ZoomWidget} from './widgets/zoom-widget';
+export {ScreenshotWidget} from './widgets/screenshot-widget';
 
-export type {FullscreenWidgetProps} from './fullscreen-widget';
-export type {CompassWidgetProps} from './compass-widget';
-export type {ZoomWidgetProps} from './zoom-widget';
-export type {ScreenshotWidgetProps} from './screenshot-widget';
+export type {FullscreenWidgetProps} from './widgets/fullscreen-widget';
+export type {CompassWidgetProps} from './widgets/compass-widget';
+export type {ZoomWidgetProps} from './widgets/zoom-widget';
+export type {ScreenshotWidgetProps} from './widgets/screenshot-widget';
+
+export type {ResetViewWidgetProps} from './widgets/reset-view-widget';
+export {ResetViewWidget} from './widgets/reset-view-widget';
 
 export type {WidgetImplProps} from './widget-impl';
 export {WidgetImpl as _WidgetImpl} from './widget-impl';
-import * as _components from './components';
-export {_components};
 
-export * from './themes';
+export {IconButton, ButtonGroup, GroupedIconButton} from './components';
```

Comment:
Note: barrel exports are strongly objected to by javascript tool builder community, they apparently substantially slow down builds, tests etc.